src/clj/witan/send/domain/academic_years.clj

```diff
@@ -1,5 +1,21 @@
 (ns witan.send.domain.academic-years)
 
+
+(defn ncy->age [ncy]
+  (+ ncy 5))
```

Comment:
So this is age at the end of the academic year (on 31st August), as [most children start school full-time in the September after their fourth birthday](https://www.gov.uk/schools-admissions/school-starting-age#:~:text=Most%20children%20start%20school%20full,start%20school%20in%20September%202022.).